rxbinding-recyclerview-v7/src/androidTest/AndroidManifest.xml

```diff
@@ -4,5 +4,6 @@
           xmlns:android="http://schemas.android.com/apk/res/android">
   <application>
     <activity android:name="com.jakewharton.rxbinding.support.v7.widget.RxRecyclerViewTestActivity"/>
+    <activity android:name="com.jakewharton.rxbinding.support.v7.widget.RxRecyclerViewChildAttachStateTestActivity"/>
```

Comment:
Is it possible to re-use `RxRecyclerViewTestActivity` but change it so that the existing recycler view tests set their own adapter (just like you have done with your tests).

If possible, I would prefer if the tests for a single component were done in a single test with a single activity. If that's not possible, it's fine. But I'd at least like to try before we jump straight to a second activity.